src/encoded/static/scss/encoded/modules/_tooltip.scss

```diff
@@ -45,9 +45,13 @@ $tooltip-color: #000;
     z-index: 1030;
     display: block;
     visibility: hidden;
-    font-size: 12px;
-    line-height: 1.4;
+    font-size: 1rem;
+    line-height: 1.2;
     opacity: 0;
+    @media screen and (max-width: $screen-md-min) {
```

Comment:
Should have `min-width` so we don’t mix mobile first and desktop first.

Reply:
Agreed. I also changed the other instance of "max-width".